### PR TITLE
#128 add create user form to admin view and adapt controller

### LINKED ISSUE #128: [admin] Add possibility to create user

### PR DESCRIPTION
**Pull request by daweedek2**
There is new table at admin view
Admin can create users.

Closes #128

### CODE REVIEW

File: src/main/resources/templates/admin.html [hunk 1]
```diff
@@ -89,6 +89,36 @@ <h4 class="header-panel" style="text-align: center">New Rating</h4>
...
+                    <input id="userName" th:field="*{userName}" />
+                    <p class="error" th:if="${#fields.hasErrors('userName')}" th:errors="*{userName}" style="color: red">UserName error.</p>
+                </div>
+                <div class="form-group">
+                    <label for="password"> Password * : </label>
+                    <input id="password" th:field="*{password}" />
+                    <p class="error" th:if="${#fields.hasErrors('password')}" th:errors="*{password}" style="color: red">Password error.</p>
+                </div>
+                <div class="form-group">
+                    <label for="birthDate"> Password * : </label>
```
Comment (mjanousek) [inline src/main/resources/templates/admin.html:115]: `<label for="birthDate"> Password * : </label>` ... copy paste mistake: birthDay~password

File: src/main/resources/templates/admin.html [hunk 2]
```diff
@@ -89,6 +89,36 @@ <h4 class="header-panel" style="text-align: center">New Rating</h4>
...
+                    <input type="date" id="birthDate" th:field="*{birthDate}" />
```
Comment (mjanousek) [inline src/main/resources/templates/admin.html:116]: probably type date is also wrong

File: src/main/resources/templates/admin.html [hunk 3]
```diff
@@ -89,6 +89,36 @@ <h4 class="header-panel" style="text-align: center">New Rating</h4>
...
+                    <p class="error" th:if="${#fields.hasErrors('birthDate')}" th:errors="*{birthDate}" style="color: red">BirthDate error.</p>
```
Comment (mjanousek) [inline src/main/resources/templates/admin.html:117]: Also there

File: src/main/resources/templates/admin.html (hunk 1 above)
Comment (daweedek2) [reply src/main/resources/templates/admin.html:115]: right, thanks

File: src/main/resources/templates/admin.html (hunk 2 above)
Comment (daweedek2) [reply src/main/resources/templates/admin.html:116]: type date is needed to get valid date String

File: src/main/resources/templates/admin.html (hunk 3 above)
Comment (daweedek2) [reply src/main/resources/templates/admin.html:117]: what is wrong here?

**Pull request merged**